API.md

```diff
@@ -110,7 +98,45 @@ If set to `True`, the cluster is destroyed on exit if there is no exception rais
 
 ### Methods
 
-#### `run_integration_tests(pytest_command, env=None)`
+#### `install_dcos_from_url(build_artifact, extra_config=None, log_output_live=False)`
+
+Installs DC/OS on the given cluster using the DC/OS advanced installation method if supported
+by the backend. This method spins up a persistent bootstrap host that supplies all
+dedicated DC/OS hosts with the necessary installation files. Since the bootstrap host is different
+from the host initating the cluster creation passing the `build_artifact` via URL string
+saves the time of copying the `build_artifact` to the boostrap host.
+
+##### `build_artifact`
+
+The HTTP(S) URL string to a build artifact to install.
+
+##### `extra_config`
+
+Configuration variables to add to a base configuration.
+
+##### `log_output_live`
+
+If set to `True`, the output of the DC/OS installation process will be logged.
```

Comment:
Is this definitely right? I thought that it was logged either way but `log_output_live=True` shows that output as it happens, rather than after it has all happened. Please re-add "To see these logs in `pytest` tests, use the `-s` flag.".